Description
This patch changes the rectangle indicating if a tag is used by a client into a bar above the tag name.
Be sure to use a font which leaves enough space between the text of the tag name and the top of the bar.
Visual: https://dwm.suckless.org/patches/activetagindicatorbar/

It would be cool if I could draw a small butterfly or something else that is cute with drw_rect.

diff -up dwm-a/dwm.c dwm-b/dwm.c
--- dwm-a/dwm.c	2019-02-02 06:55:28.000000000 -0600
+++ dwm-b/dwm.c	2019-02-23 21:43:13.359179100 -0600
@@ -719,9 +719,10 @@ drawbar(Monitor *m)
 		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeSel : SchemeNorm]);
 		drw_text(drw, x, 0, w, bh, lrpad / 2, tags[i], urg & 1 << i);
 		if (occ & 1 << i)
-			drw_rect(drw, x + boxs, boxs, boxw, boxw,
-				m == selmon && selmon->sel && selmon->sel->tags & 1 << i,
-				urg & 1 << i);
+			drw_rect(drw, x + boxw, 0, w - ( 2 * boxw + 1), boxw,
+			    m == selmon && selmon->sel && selmon->sel->tags & 1 << i,
+			    urg & 1 << i);
+
 		x += w;
 	}
 	w = blw = TEXTW(m->ltsymbol);
